src/main/java/racingcar/controller/RacingGame.java

```diff
@@ -0,0 +1,96 @@
+package racingcar.controller;
+
+import racingcar.model.Car;
+import racingcar.model.Cars;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingGame {
+    public static final String DELIMITER = ",";
+    public static Cars cars;
+    public static int trials;
+
+    public static void start() {
+        while (!initializeCars()) ;
+        while (!getTrials()) ;
+        OutputView.printRoundMessage();
+        playRounds();
+        OutputView.printWinners(findWinners());
+    }
+
+    public static void assignCars(List<Car> temporaryCars) {
+        cars = new Cars(temporaryCars);
+    }
+
+    public static Cars getCars() {
+        return cars;
+    }
+
+    public static List<Car> findWinners() {
+        return cars.getCars()
+                .stream()
+                .filter(car -> car.isMaxPosition(cars.getMaxDistance()))
+                .collect(Collectors.toList());
+    }
+
+    public static List<String> splitInput(String input) {
+        return Arrays.asList(input.split(DELIMITER));
+    }
+
+    public static void isValidNumber(String input) {
+        if (Integer.parseInt(input) < 1) {
+            throw new IllegalArgumentException("시행 횟수는 1회 이상이어야 합니다.");
+        }
+    }
+
+    private static boolean initializeCars() {
+        try {
+            OutputView.printUserPromptCarNames();
+            List<Car> temporaryCars = new ArrayList<>();
+            splitInput(InputView.askUserInput())
+                    .stream()
+                    .forEach(carName -> temporaryCars.add(new Car(carName)));
```

Comment:
> 또한, 처음 로직이 String을 인풋으로 받아서 split을 한 후에 Cars에서 그것에 대한 검증을 한 후(중복, 플레이어2명이상), Car객체들을 만들어주는 것이었습니다. 하지만 이렇게 하게 될 경우 테스트를 쓸 때의 어려움이 있어서, Car객체들을 먼저 생성 후 Cars객체를 만드는 것으로 바꾸었습니다.
이부분 때문에, 처음에 자동차를 셋팅 해줄때의(53번째줄 initializeCars 메서드) 로직이 참 에매해진 것 같습니다. (변수명도 에메해졌습니다.. temporaryCars)

테스트를 할 때 어떤 어려움이 있었던 걸까요? ㅎㅎ 
참고로 다음과 같이 작성할 수도 있어요!

```suggestion
            List<Car> cars = splitInput(InputView.askUserInput()).stream()
                    .map(Car::new)
                    .collect(Collectors.toList());
```